### ISSUE TITLE
Edge Box for a new dataset

### ISSUE THREAD

**Issue by Addhi86** [pdollar/edges#11]
Hello,

 I want to use the edge box on my dataset and compare its performance with Selective search.
In the edgeBoxesDemo.m, I have run the code with split='val' which i used the training set of Voc2007. 
The 2 graphs with the default setting like 
```
model=load('models/forest/modelBsds'); model=model.model;
model.opts.multiscale=0; model.opts.sharpen=2; model.opts.nThreads=4;

%% set up opts for edgeBoxes (see edgeBoxes.m)
opts = edgeBoxes;
opts.alpha = .65;     % step size of sliding window search
opts.beta  = .75;     % nms threshold for object proposals
opts.minScore = .01;  % min score of boxes to detect
opts.maxBoxes = 1e4;  % max number of boxes to detect
```
it shows me an output:
EdgeBoxes70  T=0.70  A=0.46  M= 804  R=0.87
Which is similar to Figure 5 (I guess). But it is mentioned in the paper that for the comparison of Edge Box with other methods you used test set.

So for the test set I have changed the training set with testing set in the \boxes folder and changed this part in the code
`split='test'; data=boxesData('split',split);`

and I am getting this output
EdgeBoxes70  T=0.70  A=0.47  M= 656  R=0.88

Therefore, I have couple of questions:

1) Why the test result is different from the paper. like According to table 2, the value of should be 800 not 656 ? Or i did something wrong

2) How can I plot the Figure 5 graph for the selective search method using VOC 2007 dataset?

3) Also, if i used the Edge box for different dataset do I need to train new model? 
Please help me to solve this

**Comment by Addhi86** [comment]
Hello, I am still waiting for the reply. 
especially for question no 2

**Comment by pdollar** [comment]
Sorry for the super late reply. You don't need to retrain the model. Just apply it to a new dataset at test time. Thanks!
